padl/dumptools/ast_utils.py

```diff
@@ -19,6 +20,13 @@ class Position:
     end_col_offset: int
 
 
+def cached_parse(source):
```

Comment:
I realize it is a very simple function, but I would appreciate a short docstring explaining what it is for.